### PR TITLE
Rewritten Release Notes section Feature/bsc#1018309

### PR DESCRIPTION
**Pull request by sven15**

### CODE REVIEW

File: xml/sle_update_upgrading.xml [hunk 1]
```diff
@@ -72,7 +72,9 @@ Check with Simona, if this is supported or needed.
     In the release notes you can find additional information on what has
     changed since the previous release of &sle;. If you are skipping one
     or more Service Packs, also check the release notes of the skipped
-    Service Packs for changes made between the different releases.
+    Service Packs for changes made between each release. The release
+    notes usually only contain the changes from the last to the current
```
Comment (sknorr) [inline xml/sle_update_upgrading.xml]: More precisely: `... usually only contain the changes between two subsequent releases.`?

**Comment by sven15** [comment]
[Add a notice to review all other release notes in case of a major migration](https://trello.com/c/DqKCoyvv/398-add-a-notice-to-review-all-other-release-notes-in-case-of-a-major-migration)

File: xml/sle_update_upgrading.xml [hunk 2]
```diff
@@ -67,26 +67,45 @@ Check with Simona, if this is supported or needed.
...
+   </itemizedlist>
+   <para>
+    The release notes also provide information that could not make it
+    into the manual on time. They also contain notes about known issues.
+   </para>
+   <para>
+    If you are skipping one or more Service Packs, check the release
+    notes of the skipped Service Packs as well. The release notes
+    usually only contain the changes from the last to the current
+    release. You can miss important changes if you are only reading the
```
Comment (sknorr) [inline xml/sle_update_upgrading.xml]: More precisely: `... usually only contain the changes between two subsequent releases.`?

Comment (sven15) [inline xml/sle_update_upgrading.xml]: Sounds better.

**Pull request merged**